scripts/bumpVersion.js

```diff
@@ -0,0 +1,46 @@
+const fs = require("fs")
+
+const filePath = "lerna.json"
+const versionBump = process.argv[2] || "patch"
+
+// Read and parse lerna.json file
+const fileData = fs.readFileSync(filePath)
+const lernaData = JSON.parse(fileData)
+
+// Get current version and split into major, minor, patch, and alpha components
+const currentVersion = lernaData.version
+const [versionWithoutPrerelease, alpha] = currentVersion.split("-")
+const [major, minor, patch] = versionWithoutPrerelease.split(".").map(Number)
+
+// Calculate new version based on specified version bump
+let newVersion = currentVersion
+switch (versionBump) {
```

Comment:
Could use `semver` here: https://www.npmjs.com/package/semver
It has lots of useful utilities, but I think the `increment` would be most useful for what we want to achieve

Reply:
Good find, I was actually looking for an existing library... Not sure how I did not find it 😓